### DISCUSSION TITLE
what is livewire/message/folder.component-name request?

### DISCUSSION THREAD

**Discussion by pixsolution** [livewire/livewire#4689] [1. Help]
Hello, on my server, doing traffic and request monitoring, I see that several Requesting always appear at the top with the following structure livewire/message/folder.component-name attached example
<img width="375" alt="a" src="https://user-images.githubusercontent.com/47191848/154466565-c578dbcd-44ec-49d5-9438-57e8a14eef9f.PNG">
the case is that I do not have any such route in my project, I mean, I don't see that there is livewire/message what can you refer to? If it exists, can it be optimized?

**Comment by silnarm** [comment]
Those are messages to the components on the back-end, to sync state & perform actions. The routes are registered in LivewireServiceProvider::registerRoutes()

If you are finding the load is unreasonably high, you probably want to review your use (or lack of!) of debounce and lazy modifiers on your model:wire attributes.
